### DISCUSSION TITLE
Hotfix for 2.3.90

### DISCUSSION THREAD

**Discussion by its-a-me-mario-the-og** [Security-Onion-Solutions/securityonion#6380] [Unsupported Versions] [answered]
In /root/soup.log I have ran into the following errors:
"Summary for local
-------------
Succeeded: 34 (changed=4)
Failed:     0
-------------
Total states run:     34
Total run time:   10.183 s

soup has been updated. Please run soup again.

The manager's pillars can be rendered. We can proceed with SOUP.

Checking to see if this is an airgap install.

Checking to see if this is a manager.

This is a manager, so we can proceed.
Found that Security Onion 2.3.90 is currently installed.

Cloning Security Onion github repo into /tmp/sogh/securityonion.
Removing previous upgrade sources.
Cloning the Security Onion Repo.
Cloning into 'securityonion'...
Verifying we have the latest soup script.
This version of the soup script is up to date. Proceeding.

Elastic License has already been accepted

Looks like you have an updated OS
Generating new repo archive
Let's see if we need to update Security Onion.
Checking to see if there are hotfixes needed
We need to apply a hotfix
You have enough space for upgrade. Proceeding with soup.
Checking for Salt Master and Minion updates.
You are already running the correct version of Salt for Security Onion.
Applying WAZUH hotfix
Detecting if ossec.conf needs corrected...
/nsm/wazuh/etc/ossec.conf does not have an XML header, so no changes are necessary.
Hotfix applied
Updating the Security Onion version file.
local:
    Data failed to compile:
----------
    Rendering SLS 'base:firewall' failed: Jinja variable 'dict object' has no attribute 'endgame'
Soup failed with error 1: Unhandled error
Unhandled error occured, please check /root/soup.log for details.

The manager's pillars can be rendered. We can proceed with SOUP.

Checking to see if this is an airgap install.

Checking to see if this is a manager.

This is a manager, so we can proceed.
Found that Security Onion 2.3.90 is currently installed.

Cloning Security Onion github repo into /tmp/sogh/securityonion.
Removing previous upgrade sources.
Cloning the Security Onion Repo.
Cloning into 'securityonion'...
Verifying we have the latest soup script.
This version of the soup script is up to date. Proceeding.

Elastic License has already been accepted

Looks like you have an updated OS
Generating new repo archive
Let's see if we need to update Security Onion.
Checking to see if there are hotfixes needed
You are already running the latest version of Security Onion.

The manager's pillars can be rendered. We can proceed with SOUP.

Checking to see if this is an airgap install.

Checking to see if this is a manager.

This is a manager, so we can proceed.
Found that Security Onion 2.3.90 is currently installed.

Cloning Security Onion github repo into /tmp/sogh/securityonion.
Removing previous upgrade sources.
Cloning the Security Onion Repo.
Cloning into 'securityonion'...
Verifying we have the latest soup script.
This version of the soup script is up to date. Proceeding.

Elastic License has already been accepted

Looks like you have an updated OS
Generating new repo archive
Let's see if we need to update Security Onion.
Checking to see if there are hotfixes needed
You are already running the latest version of Security Onion."

Also when I run so-status everything is running but the following is on WAIT_START:
so-elastalert, so-strelka-backend, so-strelka-filestream, so-strealka-frontend, and so-strelka-manager

Any help will be highly appreciative.

**Comment by TOoSmOotH** [comment, marked as answer]
This looks like your previous soup had issues to 2.3.90. 

Run the following then highstate the manager:

so-firewall addhostgroup endgame

**Reply by its-a-me-mario-the-og** [reply]
That allowed all of the services to come up as ok, but I do not have any of my dashboards or indices in Kibana.

**Reply by m0duspwnens** [reply]
Which type of node is your manager? Is it a manager, managersearch, standalone or eval? Which OS is it? Can you post the full soup.log with any sensitive information redacted? Can you verify if the following has been added / appended to your manager pillar file located under `/opt/so/saltstack/local/pillar/minions/`?

```
soc:
  es_index_patterns: '*:so-*,*:endgame-*'
```

**Reply by its-a-me-mario-the-og** [reply]
It is a heavy node, the soup log is right here:
[soup.log](https://github.com/Security-Onion-Solutions/securityonion/files/7620344/soup.log)
 the file is in /opt/so/saltstack/local/pillar/minions/ the line: soc: es_index_patterns: '*:so-*,*:endgame-*' has not been added to the file

**Reply by its-a-me-mario-the-og** [reply]
We ran `so-kibana-savedobjects-defaults` which brought back the dashboards. Still had a bunch of shards failing, but some data was somewhat there, so we blitzed the ES data with `so-elastic-clear` and now it seems to be running good.
